src/utilities/DomUtils.ts

```diff
@@ -1,3 +1,124 @@
-export function elementContains(parent: HTMLElement, child: HTMLElement): boolean {
-  return parent.contains(child);
-}
\ No newline at end of file
+
```

Comment:
I think for less redundancy and more consistency, we should generally try to avoid the words like "utility" in the filename. We can stick all utilities in a utility folder which makes the purpose clear.

So maybe this file should just be called dom.ts